client/src/containers/App/App.css

```diff
@@ -1,3 +1,24 @@
 .main-content {
   text-align: center;
 }
+.sidebar1 {
```

Comment:
If we need to add a number after our CSS it might be an indication that we need to re-think the name.  Will it work if you remove the numbers?